cmd/apps/metricsserver_app.go

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) arkade author(s) 2020. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
```

Comment:
Why did you remove this?

Reply:
Not intentionally. Don't know how I didn't notice.